R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -1451,7 +1451,7 @@ test_that("column functions", {
   expect_equal(collect(df2)[[3, 2]], TRUE)
 
   # Test that input_file_name()
-  actual_names <- sort(collect(distinct(select(df, input_file_name()))))
```

Comment:
It was sorting a DataFrame in R which isn't legitimate. In addition, there's no point of sorting. It's a single value single column DataFrame.

```
                                                                                       input_file_name()
1 file:///private/var/folders/0c/q8y15ybd3tn7sr2_jmbmftr80000gp/T/RtmpwPkEnu/sparkr-testf2872350813d.tmp
```

Reply:
Not sure if it is intentional to be a single value? Because it is `actual_names`. :)

Reply:
Oh yea, it is verified the length is 1. So removing sort looks good.